services/TipiVariable.py

```diff
@@ -23,44 +22,39 @@ class TipiVariable(object):
 
     def __init__(self, tipi_io):
         self.tipi_io = tipi_io
-        
     
     def processRequest(self, message):
-        logger.debug("request: %s", message)
-     
         # Fields:
         # caller_guid        Program's GUID
-        # action             'R', 'W', 'T'   read, write, transmit
-        # var_key            Key
-        # var_val            Value
+        # action             'R', 'W', 'U', 'T'   read, write, transmit via UDP to REMOTE_HOST, transmit via TCP to REMOTE_HOST
         # queue              Allow values to queue for these variables. (only locally for now)
-        # remote_var_key     The name of the remote variable, if we're transmitting this var/val.
-        # remote_id          Remote ID to present, if we're transmitting this var/val. We're going to use session_id for the remote_id
-
+        # results_var        results_var (optional)
+        # var_key1
+        # var_val1
+        # var_key2
+        # var_val2
+        # var_key3
+        # var_val3
+        # var_key4
+        # var_val4
         
-        # Messages look like: (Tab-delimited)
-        #
-        # CHATTI     R   REMOTE_HOST     NULL        0   NULL        NULL
-        # CHATTI     W   REMOTE_PORT     9900        0   NULL        NULL
-        # CHATTI     W   MESSAGE         [myPasswd]  0   AUTHPASS    NULL
-        # CHATTI     W   MESSAGE         TESTING123  0   MESSAGE     [session_id]
-
-
         # Now that we have message, let's parse it:
-        ti_message = message.split("\t")
-
-        caller_guid    = ti_message[0] if len(ti_message) >= 1 else ''  # Program's GUID
-        action         = ti_message[1] if len(ti_message) >= 2 else ''  # 'R', 'W', 'U', 'T'   read, write, transmit via UDP, transmit via TCP
-        var_key        = ti_message[2] if len(ti_message) >= 3 else ''  # Key
-        var_val        = ti_message[3] if len(ti_message) >= 4 else ''
-        queue          = ti_message[4] if len(ti_message) >= 5 else ''  # Allow values to queue for these variables. (only locally for now)
-        remote_var_key = ti_message[5] if len(ti_message) >= 6 else ''  # The name of the remote variable, if we're transmitting this var/val.
-        remote_id      = ti_message[6] if len(ti_message) >= 7 else ''  # Remote ID to present, if we're transmitting this var/val. We're going to use session_id for the remote_id
-
-        oled.info("VAR:%s/%s %s", caller_guid, action, var_key)
```

Comment:
that's fine, the oled stuff was neat to watch, but meaningless...   It was just the import, and this line... 
If you want to add it back in, do the import, and compose an oled message that makes sense... 
The / is a line separator... the first line uses a bigger font and can only handle about 10 characters, and the second line has a smaller font... boy,,  without one to see what it looks like, you can't really be expected to maintain that...  :)  anyway, it is just for fun....

-M@